.github/workflows/unit-tests.yaml

```
@@ -0,0 +1,82 @@
+          name: XUnit Tests
+          path: "**/dotnet-unit-test-results.trx"
+          reporter: dotnet-trx
+  
+  java-unit-tests:
+    runs-on: ['ubuntu-latest']
+    steps:
+      - name: Checkout source repository
+        uses: actions/checkout@v4
+      
+      - name: Set up JDK
+        uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)